src/components/Questions/PrevNextNav/styles.scss

```diff
@@ -0,0 +1,42 @@
+.pos-absolute {
```

Comment:
CSS classes should describe what some component is: a class could be e.g. title, title icon, description, box-header